tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -62,11 +62,27 @@ object VarDictJava extends Configuration {
   /** Path to the VarDictJava bin directory with a multitude of perl scripts. */
   val BinDir = RootDir.resolve("VarDict")
 
+  val IllegalCharacters: String = "\t "
+
   /** Pulls a sample name out of a BAM file. */
   private[vc] def extractSampleName(bam: PathToBam) : String = {
     val in = SamReaderFactory.make().open(bam)
     yieldAndThen(in.getFileHeader.getReadGroups.iterator().next().getSample) { in.close() }
   }
+
+  /** Replaces a set of illegal characters within a String that is to be used as a sample name for VarDict.
+    *
+    * @param sampleName the string that is to be used as a  sample name
+    * @param illegalCharacters the set of characters to be replaced if found, defaults to [[IllegalCharacters]]
+    * @param replacement an optional replacement character, defaulting to '_'; if [[None]] characters are just removed
+    * @return the sample name without illegal characters
+    */
+  private[vc] def replaceWhiteSpace(sampleName: String,
+                                    illegalCharacters: String = IllegalCharacters,
+                                    replacement: Option[Char] = Some('_')): String = replacement match {
+    case None    => sampleName.filter(c => !illegalCharacters.contains(c))
```

Comment:
Also, I wonder if here, or elsewhere upstream we should be `trim`ing fields, or if this problem came up in the middle of a sample name?